octavia-cli/unit_tests/test_entrypoint.py

```diff
@@ -83,17 +83,33 @@ def test_octavia_not_initialized(mocker):
     assert result.exit_code == 0
 
 
+def test_invalid_passed_headers():
+    pass
+
+
+def test_invalid_path_for_header_file():
+    pass
+
+
+def test_header_file_and_headers_flag_passing_at_the_same_time():
+    pass
+
+
+def test_header_file_properly_assignment():
+    pass
+
+
 @pytest.mark.parametrize(
-    "header_name,header_value",
-    [(None, None), ("non-empty", "non-empty"), (None, "non-empty"), ("non-empty", None)],
+    "api_headers",
+    [None, [], ["Authorization: Basic dXNlcjE6cGFzc3dvcmQ="], ["Authorization: Basic dXNlcjE6cGFzc3dvcmQ=", "Header: header_value"]],
```

Comment:
I'd love to check if an invalid header structure is passed the `get_api_client` function fails with a meaningful error. An invalid structure could be one without `:` or with multiple `:`. I made another comment suggestion to use Multi value options in Click to benefit from simpler validation. https://click.palletsprojects.com/en/8.1.x/options/#tuples-as-multi-value-options